lib/elasticsearch/drain/autoscaling.rb

```diff
@@ -41,6 +45,17 @@ def describe_instances
         @instances = instances
       end
 
+      def describe_autoscaling_group
+        group = []
+        groups = @asg_client.describe_auto_scaling_groups(
+          auto_scaling_group_names: [asg]
+        )
+        groups.auto_scaling_groups.each do |page|
```

Comment:
I am only getting one ASG but this api doesn't have the typical pager.  I guess I was using this the same way when I really don't need to.